flocker/acceptance/test_api.py

```diff
@@ -283,9 +313,13 @@ class DatasetAPITests(TestCase):
     """
     Tests for the dataset API.
     """
-    def test_dataset_creation(self):
+    def _test_create(self):
```

Comment:
:question: I like the `_foo_test` naming convention for these so that the name is more than a single `_` away from being a valid test name the runner will discover.